test/.eslintrc.json

```diff
@@ -2,5 +2,10 @@
   "env": {
     "mocha": true
   },
-  "extends": "../.eslintrc.json"
+  "extends": "../.eslintrc.json",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "experimentalObjectRestSpread": true
```

Comment:
starting with *just* test files, since that's the one place I've definitely used it so far.  I'll move it up a level once `lib/` starts to do it.